### DISCUSSION TITLE
.clear() without resetting camera

### DISCUSSION THREAD

**Discussion by sampotter** [pyvista/pyvista#4981] [Q&A]
Simple question: is it possible to `.clear()` a plotter without resetting the camera?

If it helps, I'm using a `BackgroundPlotter` from pyvistaqt.

**Comment by banesullivan** [comment]
Perhaps you just want `Plotter.renderer.clear_actors()`? I recommend poking around the implementation of `clear()` and perhaps creating your own. I suspect the `RemoveAllViewProps()` call is causing the reset, but not sure
